### PR TITLE
docs: working with individual results page

### PR DESCRIPTION
**Pull request by richardjgowers**
addresses #745

### CODE REVIEW

**Comment by hannahbaumann** [comment]
@richardjgowers : Are you also planning on talking about the additional metrics we're getting, like the overlap matrix, forward-backward analysis or the replica exchange plots? I started explaining some of that in the `details of specific protocols` section, but then wondered if it's better places here.

**Comment by richardjgowers** [comment]
@hannahbaumann not on this page/PR, but if you leave it as a todo on your PR I'll fill in the details for HREX

File: docs/guide/results/index.rst [hunk 1]
```diff
@@ -3,6 +3,54 @@
...
-contact us on our `issue tracker <https://github.com/OpenFreeEnergy/openfe/issues>`_
-if you have any questions about analysing your results!
+With :ref:`execution of your calculations <userguide_execution>` completed,
+we can now start looking at what has been produced.
+At the very least, Protocols will produce estimates of free energy differences of the ``ChemicalSystem`` \s
+(with the exception of MD Protocols which just simulate the behaviour of a single system).
+Beyond this, the exact data produced by a given Protocol can vary significantly,
+for example the HREX protocol will produce graphs to assess the quality of the simulation,
+while the MD Protocol will produce a trajectory file for the simulated system.
+For exact details on what is produced consult the :ref:`pages for each Protocol<userguide_protocols>`.
```
[the remark this answers is not in the document]
Comment (richardjgowers) [reply docs/guide/results/index.rst:13]: this is #792 but doesn't belong on this page

File: docs/guide/results/index.rst [hunk 2]
```diff
@@ -3,6 +3,54 @@
...
+
+From the command line
+---------------------
+
+If you had executed your calculation using the :ref:`quickrun <cli_quickrun>` command,
+then a ``.json`` results log file as well as a directory of files will have been produced.
+Most importantly, the ``.json`` results file has ``estimate`` and ``uncertainty`` keys,
+which serve the same purpose as the ``get_estimate()`` and ``get_uncertainty()`` methods described above.
+
+The ``json`` results file can be loaded into a Python session as::
```
[the remark this answers is not in the document]
Comment (richardjgowers) [reply docs/guide/results/index.rst]: I've clarified the headings, this section is how to work with results you generated from the cli, which probably requires Python again to load this file

File: docs/guide/results/working_with_networks.rst
[the remark this answers is not in the document]
Comment (richardjgowers) [reply docs/guide/results/working_with_networks.rst:1]: I added a real index and made this page a child of it

File: docs/guide/results/index.rst [hunk 3]
```diff
@@ -3,6 +3,56 @@
...
+From Python execution
+---------------------
+
+Executing a :class:`.ProtocolDAG` using :func:`openfe.execute_DAG` will produce a :class:`.ProtocolDAGResult`,
+representing a single iteration of estimating the free energy difference.
+One or more of these can be put into the ``.gather()`` method of the ``Protocol`` to form a :class:`.ProtocolResult`,
+this class takes care of the averaging and concatenation of different iterations of the estimation process.
+This ``ProtocolResult`` class has ``.get_estimate()`` and ``.get_uncertainty()`` methods which return the estimates
+of free energy difference along with its uncertainty.
+
```
[the remark this answers is not in the document]
Comment (IAlibay) [reply docs/guide/results/index.rst]: Raised https://github.com/OpenFreeEnergy/openfe/issues/813

**Review by IAlibay — APPROVED** [review]

**Pull request merged**